config.m4

```diff
@@ -342,6 +386,32 @@ if test "$PHP_MONGODB" != "no"; then
       PHP_MONGODB_ADD_BUILD_DIR([src/libmongoc/src/zlib-1.2.11/])
       AC_CONFIG_FILES([${ac_config_dir}/src/libmongoc/src/zlib-1.2.11/zconf.h])
     fi
+
+    if test "$PHP_CLIENT_SIDE_ENCRYPTION" = "yes"; then
+      AC_SUBST(MONGOCRYPT_ENABLE_TRACE, 1)
+      AC_SUBST(MONGOCRYPT_IS_POSIX, 1)
+      AC_SUBST(MONGOCRYPT_IS_WIN, 0)
```

Comment:
Per my [comment in MONGOCRYPT-218](https://jira.mongodb.org/browse/MONGOCRYPT-218?focusedCommentId=2653755&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2653755), I think these are obsolete. It looks like we can always compile the `os_posix/` and `os_win` sources.

Reply:
I believe so, I forgot to drop these when I updated to the latest libmongocrypt version.